packages/documentation/src/stories/components/forms/select/select.stories.ts

```diff
@@ -242,7 +242,7 @@ const Template: Story = {
       args.validation === 'is-invalid'
         ? html` <p class="invalid-feedback">Eraro okazis!</p> `
         : null,
-      args.hint !== '' ? html` <div class="form-text">${args.hint}</div> ` : null,
+      args.hint !== '' ? html` <div class="form-hint">${args.hint}</div> ` : null,
```

Comment:
Also connect this on the input element via aria-describedby.

```suggestion
      args.hint !== '' ? html` <p class="form-hint" id="form-hint-example">${args.hint}</p> ` : null,
```